### PR TITLE
Create issues template

### PR DESCRIPTION
**Pull request by nnagewad**
# Context
Based off of a conversation @ChristineWuerth and I had to simplify the creation of issues, particularly on components, patterns and frontend templates that still need to be documented and added to the GOV.UK Publishing Design Guide.

For this process, I've referred to the following articles:
- [Manually creating a single issue template for your repository](https://docs.github.com/en/enterprise-cloud@latest/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository)
- [Syntax for GitHub's form schema](https://docs.github.com/en/enterprise-cloud@latest/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)
- [Configuring issue templates for your repository](https://docs.github.com/en/enterprise-cloud@latest/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/01-new-components.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: New component
+description: Document a component that needs to be added to the GOV.UK Publishing Design Guide
+title: "Document [Enter the name of the component]"
```
Comment (ChristineWuerth) [inline .github/ISSUE_TEMPLATE/01-new-components.yml:3]: What's the difference between name and title?

File: .github/ISSUE_TEMPLATE/01-new-components.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+labels: ["component", "new documentation"]
+projects: ["alphagov/99"]
```
Comment (ChristineWuerth) [inline .github/ISSUE_TEMPLATE/01-new-components.yml:5]: should this not be the design guide?

File: .github/ISSUE_TEMPLATE/01-new-components.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      value: "# Steps to start documenting a new component"
+  - type: markdown
+    attributes:
+      value: |
+        - Assign yourself to this issue
+        - Under the "Development" select the "Create a new branch" to start documenting the new component
+            - Be sure to check the following criterias:
+                - **Repository destination** is set to `alphagov/govuk-design-guide`
+                - **Branch source** is set to `main`
+            - Select "Checkout locally", which will automatically create a new branch related to this issue
```
Comment (ChristineWuerth) [inline .github/ISSUE_TEMPLATE/01-new-components.yml:18]: Do we want branches already created for each issue?

File: .github/ISSUE_TEMPLATE/01-new-components.yml (hunk 2 above)
Comment (nnagewad) [reply .github/ISSUE_TEMPLATE/01-new-components.yml:5]: If you're referring to `projects` this refers to the [Community Board](https://github.com/orgs/alphagov/projects/99/).

The way the syntax works is `[name of user or organization]`/`[board number, which can be found in the board's URL]`

File: .github/ISSUE_TEMPLATE/01-new-component.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      value: |
+        - Assign yourself to this issue
+        - Under the "Development" select the "Create a new branch" to start documenting the new component
+            - Be sure to check the following criterias:
+                - **Repository destination** is set to `alphagov/govuk-design-guide`
+                - **Branch source** is set to `main`
+            - Select "Checkout locally", which will automatically create a new branch related to this issue
+            - Once selected, copy the command and run it in your VS Code terminal
+            - Confirm that you are working on the new branch by selecting the branch icon in the bottom left corner of VS Code
+        - Start documenting the new component be sure to duplicate the template file `*component-documentation-template` and rename it to the component
```
Comment (ChristineWuerth) [inline .github/ISSUE_TEMPLATE/01-new-component.yml:21]: would it be of value to link to the template?

File: .github/ISSUE_TEMPLATE/01-new-component.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+title: "Document [Enter the name of the component]"
+labels: ["component", "new documentation"]
+projects: ["alphagov/99"]
+body:
+  - type: markdown
+    attributes:
+      value: "# Steps to start documenting a new component"
+  - type: markdown
+    attributes:
+      value: |
```
Comment (ChristineWuerth) [inline .github/ISSUE_TEMPLATE/01-new-component.yml:12]: Can we add a section for existing information/documentation to use?

**Review by ChristineWuerth — COMMENTED** [review]
Added a few thoughts and questions

File: .github/ISSUE_TEMPLATE/01-new-component.yml (hunk 4 above)
Comment (nnagewad) [reply .github/ISSUE_TEMPLATE/01-new-component.yml:21]: Na... From that point the individual should be working in a code editor like VS Code.

**Pull request merged**